### PR TITLE
Create mohdanss.json

### PR DESCRIPTION
**Pull request by mohdanss**

### CODE REVIEW

File: Submissions/mohdanss.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "Muhammad Ans",
+  "img": "https://avatars.githubusercontent.com/u/58670177?v=4",
+  "email": "mohd.ans.riaz@gmail.com",
+  "links": {
+    "website": "mohdanss.github.io/portfolio",
```
Comment (ProsperousHeart) [inline Submissions/mohdanss.json:6]: please provide the full URL including http or https

**Review by ProsperousHeart — CHANGES_REQUESTED** [review]

File: Submissions/mohdanss.json (hunk 1 above)
Comment (mohdanss) [reply Submissions/mohdanss.json:6]: Updated the website url with 'https://'

**Review by mohdanss — COMMENTED** [review]
thanks!

**Review by ProsperousHeart — CHANGES_REQUESTED** [review]
please remove the nazmul file from your git - otherwise approved

**Pull request closed without merging**